src/RolloutScorer/RolloutScorer/StandardConfig.cs

```diff
@@ -58,7 +58,7 @@ public static class StandardConfig
             GithubConfig = new GithubConfig
             {
                 ScorecardsGithubOrg = "dotnet",
-                ScorecardsGithubRepo = "core-eng",
+                ScorecardsGithubRepo = "arcade",
                 ScorecardsDirectoryPath = "Documentation/Rollout-Scorecards/",
```

Comment:
Is it all right to leave it on the same path or is there a different place we want to put it? (Maybe near the dev guide?)

Reply:
Oh good idea. I'll move them to the TeamProcess directory. Thoughts on migrating the old cards to arcade?

Reply:
Let's migrate them. Even if the links in the OneNote break, we'll still have them all together.

Reply:
PR incoming.